rfcs/SemanticNullability.md

```diff
@@ -316,6 +316,9 @@ The implementation required to make the proposal work should be simple.
 
 Criteria score: 🥉
 
+* ✂️ Objection: proposal to increase the score to 🥈. Simplicity is a defining characteristic of
+a language and is vital to adoption.
```

Comment:
Hard agree; simplicitly is definitely a nice-to-have - it's one of the [guiding principles](https://github.com/graphql/graphql-spec/blob/main/CONTRIBUTING.md#guiding-principles)!

```suggestion
Criteria score: 🥈
```